app/views/idv/shared/_ssn.html.erb

```diff
@@ -1,6 +1,6 @@
 <% title t('titles.doc_auth.ssn') %>
```

Comment:
one idea, something we've done in other partials that take in a few local variables is a comment up top that lists them all, which can make it easier to integrate into new spots. Not a requirement but something we might consider doing since we're here

https://github.com/18F/identity-idp/blob/3afaa2739707a07b3c7d0424078a46923e431470/app/views/shared/_one_time_code_input.html.erb#L1-L11

Reply:
Added in da36d11064149805774a30684df69fbc3e1a06aa